.github/workflows/codecov.yml

```diff
@@ -0,0 +1,51 @@
+on:
+  # push:
+  #   branches: [main, 'release-v**']
+  pull_request:
+name: Code coverage
+concurrency:
+  group: 'codecov-${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+permissions:
+    id-token: write
+    contents: read
+jobs:
+  codecov:
+    runs-on: ubuntu-latest-8-cores
+    # container:
+    #   image: xd009642/tarpaulin:develop-nightly
+    #   options: --security-opt seccomp=unconfined    
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Prep build
+        uses: ./.github/actions/prep-ubuntu
+        with:
+          cache: disabled
+
+      - name: Install Tarpaulin
+        run: cargo install --locked cargo-tarpaulin
+        
+      - name: generate codecov reports
+        env:
+          RUST_BACKTRACE: 1
+        run: |
+          # cargo tarpaulin  --release  --features fast-runtime --fail-under 10 \
+          #  --no-fail-fast --workspace --timeout 3600 --engine llvm \
+          #  --exclude-files **/mock.rs **/weights.rs **/weights/* --out xml --out html
+          cargo tarpaulin --timeout 3600 -e runtime-integration-tests --features fast-runtime --workspace -v --exclude-files **/mock.rs **/weights.rs **/weights/* --out xml
```

Comment:
We might have a winning combination. Excluding integration tests makes the job last 40m and it's now green

Reply:
Great!

As a tech debt regarding this. We need to evaluate if excluding this affect somehow to the statistics of the coverage. If such statistics are by crate, then not. If they're computed by workspace then yes.